.github/workflows/python-run-tests-test.yml

```diff
@@ -5,11 +5,16 @@ name: python-run-tests test workflow
 on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+  # schedules this to run weekly
+  schedule:
+    - cron: '5 3 * * 0' # At 03:05 on Sunday
```

Comment:
comment on line 1 states run each day, cron runs once a week (which is probably enough)

Reply:
Good point, I'll fix the comment.

The requirement on the ticket is daily, but when it came to creating the cron arguments I thought weekly was sufficient.

@chrisbc are you happy with a weekly schedule?

Reply:
yes, I think that's enough to pick up upstream changes, they're usually well forecast. We probably will need to check this manually for new warning messages.